src/common/types/store-data/unified-data-interface.ts

```diff
@@ -37,6 +37,7 @@ export interface PlatformData {
 export interface TargetAppData {
     name: string;
     url?: string;
+    device?: string;
```

Comment:
This seems like it should not be here ... maybe have it at the ScanMetadata level (or another better fitting object that has this)?

Reply:
I'm just thinking/saying that since target app data should contain data about the targeted application whereas device would be above that. If we do want to put this here I would heavily recommend re-naming to something else that seems to encompass all concepts?